hw-model/src/xi3c/controller.rs

```diff
@@ -226,6 +226,10 @@ impl Controller {
         Ok(())
     }
 
+    pub fn i3c_not_ready(&self) {
+        self.ready.set(false);
+    }
```

Comment:
```suggestion
    pub fn set_i3c_not_ready(&self) {
        self.ready.set(false);
    }
```

nit: `i3c_not_ready` sounds like a boolean